test/browser-interface.test.js

```diff
@@ -19,3 +20,59 @@ test('errors early if access token not provided', () => {
     }
   );
 });
+
+describe('test node progress events', () => {
+  let request;
+  const { mockToken } = tu;
+
+  afterEach(() => {
+    mockXHR.teardown();
+  });
+
+  beforeEach(() => {
+    mockXHR.setup();
+    const accessToken = mockToken();
+    mockXHR.get(
+      `https://api.mapbox.com/styles/v1/mockuser/foo?access_token=${accessToken}`,
+      {
+        status: 200,
+        headers: {
+          'Content-Length': '18',
```

Comment:
Would you mind adding a comment with the URL of the mockXhr docs that explain why this works? I expect I'd be confused by this if I look at it again 4 months later.